src/convert.ts

```diff
@@ -148,6 +153,12 @@ function mh2bytes (hash: string) {
   return uint8ArrayConcat([size, mh], size.length + mh.length)
 }
 
+function mb2bytes(mbstr: string) {
+  let mb = MB.decode(mbstr)
```

Comment:
do we need to handle the case where decoding the multibase fails?

Reply:
`decode` would throw in some failure cases

https://multiformats.github.io/js-multibase/#decode
https://github.com/multiformats/js-multibase/blob/dde00682ed1a2539677c1e2b17dc6a3c112c8e3a/src/base.js#L40-L47

That said I don't think multibase is aware of what kind of data it's decoding, so it should not be able to detect things like the field being truncated. I don't think the existing analogous multihash functions do validation of this kind in all cases either, though.

https://github.com/multiformats/js-multiaddr/blob/master/src/convert.ts#L140-L142